modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,80 @@ definition {
 		}
 	}
 
+	@description = "LPS-171625 - Verify that after updating a system object entry, triggered email notifications using Object Author and Current User terms are filled correctly for the To, From Address, From Name, CC, BCC and Template Body fields"
+	@priority = 5
+	test CanUseObjectAuthorAndCurrentUserTermsInEmailNotificationOnSystemObject {
+		property custom.properties = "feature.flag.LPS-171625=true";
+		property portal.acceptance = "true";
+
+		task ("Given an email notification template with the applicable fields filled with the Author and Current User terms") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "[%CURRENT_USER_EMAIL_ADDRESS%]",
+				fromAddress = "[%USER_AUTHOR_FIRST_NAME%]",
+				fromName = "[%CURRENT_USER_EMAIL_ADDRESS%]",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "[%USER_AUTHOR_FIRST_NAME%]");
```

Comment:
Here can you use the _[%USER_AUTHOR_LAST_NAME%]_ instead the [%USER_AUTHOR_FIRST_NAME%] again, just to increase the coverage using a different term.